### DISCUSSION TITLE
Is there a changelog for this project?

### DISCUSSION THREAD

**Discussion by heloineto** [amannn/next-intl#645] [Q&A] [answered]
Hi there, congratulations on the amazing project!

If there isn't a changelog already, It would be very helpful to use the [Releases](https://github.com/amannn/next-intl/releases) feature of GitHub.

Next.js 13 came with a lot of changes, and I feel everyone is still trying to figure things out.
As such, I imagine there will be a lot of changes to this package, and it would be very important to know when things change.

**Comment by amannn** [comment, marked as answer]
Currently the changes are documented in the changelog: https://github.com/amannn/next-intl/blob/main/CHANGELOG.md
